apps/challenges/migrations/0069_add_challenge_phase_prioritize_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
change the file name to -- `0069_add_boolean_field_to_prioritize_phase_submissions.py `